pset-functions.js

```diff
@@ -29,8 +29,41 @@ Use the following test cases to confirm your program meets the success criteria
 ************/
 console.log('Problem 1:')
 
-// Add your code below this line
+let playerCardScore = 21
+let dealerCardScore = 21
+
+const maximumScore = 21
+const overScore = 22
+const defaultScore = 0
+
+function blackJack(playerCardScore, dealerCardScore) {
```

Comment:
🛠  Read the comment below and update your code accordingly

@Defy1x the way the function is written it's dependent on variables "outside" of the function such as `defaultScore`, `overScore`, `maximumScore`. Functions are supposed to be written in a way that allows them to be easily reused. 

An easy fix, would be to move these variables "inside" the function and make them "local" to the function.

Reply:
Done.